build/RuntimeStore.targets

```diff
@@ -0,0 +1,176 @@
+<Project>
+  <Import Project="common.props" />
+
+  <PropertyGroup>
+    <_DependencyBuildDirectory>$(RepositoryRoot).deps\build\</_DependencyBuildDirectory>
+    <_BuildScriptsDirectory>$(MSBuildThisFileDirectory)tools\scripts\</_BuildScriptsDirectory>
+    <_WorkRoot>$(RepositoryRoot).w\</_WorkRoot>
+    <_RuntimeStoreWorkDirectory>$(_WorkRoot).rw\</_RuntimeStoreWorkDirectory>
+    <_RuntimeStoreOutputDirectory>$(_WorkRoot).ro\</_RuntimeStoreOutputDirectory>
+    <_TemplatesDirectory>$(MSBuildThisFileDirectory)tools\templates\</_TemplatesDirectory>
+    <_SrcDirectory>$(RepositoryRoot)src\</_SrcDirectory>
+    <_AllMetapackageDirectory>$(_SrcDirectory)Microsoft.AspNetCore.All\</_AllMetapackageDirectory>
+    <_AllMetapackageProject>$(_AllMetapackageDirectory)Microsoft.AspNetCore.All.csproj</_AllMetapackageProject>
+    <_ExistingManifestsDirectory>$(_AllMetapackageDirectory)build\</_ExistingManifestsDirectory>
+    <_ArtifactsZipDirectory>$(ArtifactsDir)zip\</_ArtifactsZipDirectory>
+    <_StoreZipDirectory>$(_ArtifactsZipDirectory)rs\</_StoreZipDirectory>
+    <_SymbolsZipDirectory>$(_ArtifactsZipDirectory)symbols\</_SymbolsZipDirectory>
+    <_DepsOutputDirectory>$(ArtifactsDir)deps\</_DepsOutputDirectory>
+  </PropertyGroup>
+
+  <Target Name="BuildAllMetapackage" DependsOnTargets="ResolveRepoInfo">
+    <!-- Generate RS.Reference project -->
+    <RepoTasks.AddMetapackageReferences
+      ReferencePackagePath="$(_AllMetapackageProject)"
+      BuildArtifacts="@(ArtifactInfo)"
+      PackageArtifacts="@(PackageArtifact)" />
+
+    <!-- Restore and pack-->
+    <MSBuild Projects="$(_AllMetapackageProject)"
+      Targets="Restore;Pack"
+      Properties="Configuration=Release" />
+  </Target>
+
+  <Target Name="UpdateNuGetConfig">
+    <UpdatePackageSource
+      NuGetConfigPath="$(RepositoryRoot)NuGet.config"
```

Comment:
Can we either make a non-tracked copy of this file or use MSBuild properties instead? I'd like to avoid the UpdatePackageSource task going forward because it makes changes to a file tracked by git. I've mistakenly committed the changes it makes a few times now.

If you want to use MSBuild props, these properties can be used to control the feeds used to restore a project:

RestoreSources
RestoreFallbackFolders
RestoreAdditionalProjectSources
RestoreAdditionalProjectFallbackFolders
RestoreConfigFile

Reply:
I've moved the nuget.config to the work directory. We can't really just pass in properties because this is required for dotnet-store which looks for a NuGet.config inside the working dir.